src/App.tsx

```diff
@@ -1,9 +1,25 @@
 import './App.scss';
+import {
+  HashRouter as Router, Navigate, Route, Routes,
+} from 'react-router-dom';
+import { PeoplePage } from './components/PeoplePage';
+import { Header } from './components/Header';
 
-const App = () => (
-  <div className="App">
-    <h1>People table</h1>
-  </div>
-);
+const App = () => {
+  return (
+    <Router>
+      <div className="App">
+        <Header />
+
+        <Routes>
+          <Route path="/" element={<h1>Homepage</h1>} />
+          <Route path="/people" element={<PeoplePage />} />
+          <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
`replace`?
here is why – https://reach.tech/router/api/navigate
also – this one
![Screen Shot 2022-08-05 at 09 31 42](https://user-images.githubusercontent.com/75496546/183015874-4febb8dc-7972-4e73-a320-65a2a52a4aca.png)